base/special/bessel.jl

```diff
@@ -370,7 +378,14 @@ for f in ("i", "ix", "j", "jx", "k", "kx", "y", "yx")
     bfn = symbol("bessel", f)
     @eval begin
         $bfn(nu::Real, z::Complex64) = Complex64($bfn(Float64(nu), Complex128(z)))
-        $bfn(nu::Real, z::Complex) = $bfn(Float64(nu), Complex128(z))
+        $bfn{T}(nu::Real, z::Complex{T}) = if T === BigInt
+			throw(ArgumentError("Bessel is currently not supported for type Complex{BigInt} (see #11512)"))
```

Comment:
Note the style guidelines in CONTRIBUTING.md specify 4-space indent, not tabs. It's also preferable to use `function` syntax once the definition requires multiple lines.